marklogic-data-hub-central/ui/src/components/entities/custom/custom-card.tsx

```diff
@@ -41,9 +40,17 @@ const CustomCard: React.FC<Props> = (props) => {
                     <Card
                         actions={[
                             <span></span>,
-                            <MLTooltip title={'Settings'} placement="bottom"><Icon type="setting" key="setting" role="settings-custom button" data-testid={elem.name+'-settings'} onClick={() => OpenCustomSettingsDialog(index)}/></MLTooltip>,
-                            <MLTooltip title={'Edit'} placement="bottom"><Icon type="edit" key="edit" role="edit-custom button" data-testid={elem.name+'-edit'} onClick={() => OpenCustomDialog(index)}/></MLTooltip>,
-                            <i role="disabled-delete-custom button" onClick={(event) => event.preventDefault()}><FontAwesomeIcon icon={faTrashAlt} className={styles.disabledDeleteIcon} size="lg"/></i>,
+                            <MLTooltip title={'Settings'} placement="bottom">
+                                <SettingOutlined key="setting" role="settings-custom button" data-testid={elem.name+'-settings'} onClick={() => OpenCustomSettingsDialog(index)} />
```

Comment:
@wooldridge do these roles look right to you.
role="button" is something that accessibility/screen readers can decipher.